quiche/src/lib.rs

```diff
@@ -8029,21 +8822,37 @@ pub mod testing {
 
         let epoch = hdr.ty.to_epoch()?;
 
-        let aead = conn.pkt_num_spaces[epoch].crypto_open.as_ref().unwrap();
+        let aead = conn
+            .pkt_num_spaces
+            .crypto
+            .get(epoch)
+            .crypto_open
+            .as_ref()
+            .unwrap();
 
         let payload_len = b.cap();
 
         packet::decrypt_hdr(&mut b, &mut hdr, aead).unwrap();
 
         let pn = packet::decode_pkt_num(
-            conn.pkt_num_spaces[epoch].largest_rx_pkt_num,
+            conn.pkt_num_spaces.spaces.get(epoch, 0)?.largest_rx_pkt_num,
             hdr.pkt_num,
             hdr.pkt_num_len,
         );
 
-        let mut payload =
-            packet::decrypt_pkt(&mut b, pn, hdr.pkt_num_len, payload_len, aead)
-                .unwrap();
+        // We don't support multipath in this method.
```

Comment:
Not supporting this is pretty unfortunate. I think this would be useful for tests. What would it take to support with with multipath?

Reply:
To be more specific, I'd like to use this for implementing a test case for stream/path affinity: https://github.com/ehaydenr/quiche/commit/9f71ed79eab4056e213418ff86ca2450c96dc038#diff-2ac678416f8d7ea462a714681b63acedcbbb7f2c99e537ce61b7667b3bf4ac41R16631

Reply:
Oh right, I did not implement that before since I did not need that, but actually providing Multipath support in `decode_pkt()` is not very hard (just pushed a7b1ff8dcdf3d182dd51ffa354b8c9cf5fd8878a to fix that).

Reply:
Hm I tried using `decrypt_pkt`, but it panics on `find_scid_seq` being `None`. https://github.com/ehaydenr/quiche/commit/d7da76eccdc54bea8ce588d201f140d34752d6f9 🤔

Reply:
I think the issue is elsewhere, as with some minor changes I make the test passes :-) See my comments on your commit.